src/index.js

```diff
@@ -4,18 +4,24 @@ const { createElement } = require('react');
 const { renderToString } = require('react-dom/server');
 const { ServerStyleSheet } = require('styled-components');
 
-const page = (title, styles, body) => `<!DOCTYPE html>
+const h = createElement;
+
+const page = (title, styles, body, bundle, props) => `<!DOCTYPE html>
 <html>
   <head>
     <title>${title}</title>
     ${styles}
   </head>
   <body style="margin: 0;">
-    ${body}
+    <div id="root">
+      ${body}
+    </div>
+    ${props ? `<script>window.appProps = ${JSON.stringify(props).replace(/</g, '\\u003c')};</script>`: ''}
```

Comment:
How about `hydrateProps` or something along the lines of suggesting they're specifically for hydration (and not for other use)?

Reply:
Yeah, I was thinking of renaming it. Not sure what to go for though. I saw an article that used `window.__PRELOADED_STATE__`.

How about `.preloadedProps`, `.staticProps`, `.initialProps` or `.hydrationProps`?

Reply:
I think my favourite there is `hydrateProps` because it's super honest and obvious!

Reply:
Let's go with that then.

Reply:
Updated.